trade-finance-manager-ui/styles/block-header.scss

```diff
@@ -0,0 +1,11 @@
+.block-header {
```

Comment:
How does the case summary have a blue background etc, can we re-use anything from there?

Reply:
The base styling looks fairly similar - I've commonised this selector (`block-header`) and added a new selector `fee-record-summary` which extends this, and also updated `case-summary` to also extend this commonised header